inc/load.php

```diff
@@ -82,6 +82,7 @@ function load_autoload($name){
         'RemoteAPI'             => DOKU_INC.'inc/remote.php',
         'RemoteAPICore'         => DOKU_INC.'inc/RemoteAPICore.php',
         'Subscription'          => DOKU_INC.'inc/subscription.php',
+        'StyleUtil'             => DOKU_INC.'inc/StyleUtil.php',
```

Comment:
This shouldn't be needed because the file is namespaced and the autoloader should take care of this.

Reply:
Also the class seems to be named ``StyleUtils``?